libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4776,19 +4776,21 @@ void GCS_MAVLINK::send_rpm() const
         return;
     }
 
-    if (!rpm->enabled(0) && !rpm->enabled(1)) {
+    if (!rpm->enabled(0) && !rpm->enabled(1) && !rpm->enabled(2)) {
         return;
     }
 
-    float rpm1 = -1, rpm2 = -1;
+    float rpm1 = -1, rpm2 = -1, rpm3 = -1;
 
     rpm->get_rpm(0, rpm1);
     rpm->get_rpm(1, rpm2);
+    rpm->get_rpm(3, rpm3);
 
     mavlink_msg_rpm_send(
         chan,
         rpm1,
-        rpm2);
+        rpm2,
+        rpm3);
```

Comment:
You can't just add a new field to a mavlink message like this.  We would either need to update the message definition to have a third field. Or move to an instance based message definition.

```suggestion
        rpm3);
```